subprojects/packagefiles/libgrapheme/meson.build

```diff
@@ -0,0 +1,75 @@
+project(
+  'libgrapheme',
+  'c',
+  version: '2.0.2',
+  default_options: [
+    'c_std=c99',
+    'warning_level=3',
+    'b_lundef=false',
+    'buildtype=minsize',
+  ],
+  meson_version: '>=1.4.0',
```

Comment:
At least according to Meson's warnings (which aren't always complete) this can be `>=0.57.0`.